### PR TITLE
rename 'priority' to 'unsolved' and remove 'unsolved' option

### PR DESCRIPTION
**Pull request by pcvera**
per feedback from playtest

### CODE REVIEW

**Comment by rgossiaux** [comment]
What if instead of removing this option we have a checkbox that only appears when you're in "UNSOLVED" that's like "Include puzzles with solved metas" or something? That way both options are still available and it's clear what everything does.

**Comment by pcvera** [comment]
implemented @rgossiaux 's suggestion!

File: hunts/src/HuntViewMain.js [hunk 1]
```diff
@@ -169,28 +169,36 @@ export const HuntViewMain = (props) => {
...
-            ></input>
             Unsolved
           </label>
+          {(filterSolved === SOLVE_STATE_FILTER_OPTIONS.PRIORITY ||
+            filterSolved === SOLVE_STATE_FILTER_OPTIONS.UNSOLVED) && (
+            <label>
+              <input
+                style={{ margin: "0 5px 0 10px" }}
+                type="checkbox"
+                checked={filterSolved === SOLVE_STATE_FILTER_OPTIONS.UNSOLVED}
```
Comment (asdfryan) [inline hunts/src/HuntViewMain.js:190]: Do you mind renaming PRIORITY->UNSOLVED and UNSOLVED->UNSOLVED_WITH_SOLVED_METAS or something equivalent? (also add comments with more details on what they mean, perhaps with an example)

**Review by asdfryan — APPROVED** [review]
Thank you! Minor nit on renaming the state + adding comments.

**Review by rgossiaux — APPROVED** [review]
Thank you!!

**Pull request merged**